server/fishtest/templates/tests_view.mak

```diff
@@ -294,10 +294,30 @@ from fishtest.util import worker_name
             % endif
             </td>
             <td>
+            <%
+	      if 'worker_info' in task:
+	        if 'python_version' in task['worker_info']:
+                  gcc_version = ".".join([str(m) for m in task['worker_info']['gcc_version']])
+                  python_version = ".".join([str(m) for m in task['worker_info']['python_version']])
+                  version = task['worker_info']['version']
+                # The code below is for backward compatibility.
+                # To be able to delete it we have to convert the database.
+                # This would be a good idea anyway since it would allow us to delete a lot of cruft
+                # from the server code.
+                # Let's first see if this code works well with old db
+                # entries.
+                else:
+                  gcc_version = task['worker_info'].get('gcc_version', '?')
+                  version = task['worker_info'].get('version','?:?')
```

Comment:
DEV fixed to deal with tests having only the worker version (`get()` returns a number):
```python
                  version = str(task['worker_info'].get('version','?:?')).split(":")
                  python_version = "?" if len(version) == 1 else version[1]
                  version = version[0]
```
https://dfts-0.pigazzini.it/tests/view/5a0972dd0ebc590ccbb8a9fa
https://tests.stockfishchess.org/tests/view/5a0972dd0ebc590ccbb8a9fa